WonSeungHyun/.gitignore

```
@@ -0,0 +1,93 @@
+/.env
+/node_modules/
+db/schema.sql
```

아래와 같이 수정해 주시고 `git rm -rf --cached .` 명령어를 통해 캐시를 삭제해 주세요 !
이후 
`git add . `
`git commit -m "메세지"`
`git push ~해당브랜치`
를 통해서 다시 수정해 주세요!
```suggestion
schema.sql
```